src/utilities/utilities.ts

```diff
@@ -56,12 +95,16 @@ export async function execFileStreamOutput(
     stderr: Stream.Writable | null,
     token: vscode.CancellationToken | null,
     options: cp.ExecFileOptions = {},
-    folderContext?: FolderContext
+    folderContext?: FolderContext,
+    customSwiftRuntime = true
 ): Promise<{ stdout: string; stderr: string }> {
     folderContext?.workspaceContext.outputChannel.logDiagnostic(
         `Exec: ${executable} ${args.join(" ")}`,
         folderContext.name
     );
+    if (configuration.runtimePath.length > 0 && customSwiftRuntime) {
```

Comment:
Do we need to check `configuration.runtimePath.length` here given the same check is in `swiftRuntimeEnv`

Reply:
This will prevent us from initializing `"env": {}`

Reply:
ok fair enough